src/app/app-extras.module.ts

```diff
@@ -13,6 +15,12 @@ require('style-loader!prismjs/themes/prism.css');
   providers: [
     StacheTitleService,
     StacheNavService
+  ],
+  declarations: [
+    StacheStickToTopDirective
```

Comment:
It will be, and is *see below*   But, because of how builder works, we need to declare these things separately in the app-settings for building locally in this project.    Builder does not pull in `StacheModule`.  In this case, we have to declare the directives like we do the services to make use of them